staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/compilation.go

```diff
@@ -66,6 +66,24 @@ type CompilationResult struct {
 	MaxCost uint64
 }
 
+var baseEnv *cel.Env
+
+func init() {
```

Comment:
using `init()` means anything linking to this package will incur this cost. Would a `func getBaseEnv()` that uses a `sync.Once` to initialize be that different, cost-wise? If not, I prefer that.

Reply:
I'll actually do the same for cel-go now that you mention it. There will be a one-time initialization cost of 1.5ms or so, but after that it's ~100us per invocation.

Reply:
I'll bump the release number once this is in: https://github.com/google/cel-go/pull/520

Reply:
Good catch @liggitt, +1 to sync.Once here.

I'll pick up the bump in this PR when it's ready.

Reply:
Released as v0.11.2

Reply:
I've picked up cel 0.11.2 in this PR.